Docker for development does not work

When executing `docker compose -f docker-compose-dev.yml up` the docker container starts properly but the changes made in the source code are not visible in the frontend.
Since `lein watch` does work when executed locally, this problem doesN#t have to be solved immediatily.

functional@code review: @katauber

`docker compose -f docker-compose-dev.yml up` fails with during `./gradlew install`

@katauber Please have a look:

```
 => ERROR [ 8/12] RUN ./gradlew install                                                                                                                                                                      59.4s 
------                                                                                                                                                                                                             
 > [ 8/12] RUN ./gradlew install:                                                                                                                                                                                  
0.273 Downloading https://services.gradle.org/distributions/gradle-8.14.4-bin.zip                                                                                                                                  
0.921 .............10%.............20%.............30%.............40%.............50%.............60%.............70%.............80%.............90%..............100%                                           
6.325                                                                                                                                                                                                              
6.325 Welcome to Gradle 8.14.4!                                                                                                                                                                                    
6.326 
6.326 Here are the highlights of this release:
6.326  - Java 24 support
6.326  - GraalVM Native Image toolchain selection
6.326  - Enhancements to test reporting
6.326  - Build Authoring improvements
6.326 
6.326 For more details see https://docs.gradle.org/8.14.4/release-notes.html
6.326 
6.326 Starting a Gradle Daemon (subsequent builds will be faster)
22.83 
22.83 > Configure project :
22.83 Making snapshot build: master-SNAPSHOT
57.32 
57.32 > Task :metafacture-commons:compileJava
57.52 > Task :metafacture-framework:compileJava
57.72 > Task :metafacture-flowcontrol:compileJava
58.42 > Task :metafacture-biblio:compileJava
58.42 > Task :metafacture-biblio:processResources
58.42 > Task :metafacture-biblio:classes
58.43 > Task :metafacture-biblio:jar
58.43 > Task :metafacture-biblio:generateMetadataFileForMavenArtifactsPublication
58.43 > Task :metafacture-biblio:generatePomFileForMavenArtifactsPublication
58.82 
58.82 > Task :metafacture-biblio:javadoc FAILED
58.82 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/MabDecoder.java:36: error: unmappable character (0xC3) for encoding US-ASCII
58.82  * @author Markus Michael Geipel, Christoph B??hme
58.82                                              ^
58.82 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/MabDecoder.java:36: error: unmappable character (0xB6) for encoding US-ASCII
58.82  * @author Markus Michael Geipel, Christoph B??hme
58.82                                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Decoder.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Decoder.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Decoder.java:131: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Decoder.java:131: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21EventNames.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21EventNames.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21EventNames.java:22: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21EventNames.java:22: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Constants.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Constants.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Constants.java:27: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Constants.java:27: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Encoder.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Encoder.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Encoder.java:66: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Encoder.java:66: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaParserState.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016,2019 Christoph B??hme and others
58.83                                   ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaParserState.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016,2019 Christoph B??hme and others
58.83                                    ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaParserState.java:32: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaParserState.java:32: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaParserContext.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaParserContext.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaParserContext.java:29: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaParserContext.java:29: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaMultiscriptRemodeler.java:100: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaMultiscriptRemodeler.java:100: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaDecoder.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016, 2019 Christoph B??hme and others
58.83                                    ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaDecoder.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016, 2019 Christoph B??hme and others
58.83                                     ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaDecoder.java:146: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaDecoder.java:146: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaConstants.java:1: error: unmappable character (0xC3) for encoding US-ASCII
58.83 /* Copyright 2016,2019 Christoph B??hme and others
58.83                                   ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaConstants.java:1: error: unmappable character (0xB6) for encoding US-ASCII
58.83 /* Copyright 2016,2019 Christoph B??hme and others
58.83                                    ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaConstants.java:23: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme (initial implementation)
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/pica/PicaConstants.java:23: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme (initial implementation)
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/DirectoryEntry.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/DirectoryEntry.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/DirectoryEntry.java:25: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/DirectoryEntry.java:25: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso2709Constants.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso2709Constants.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso2709Constants.java:23: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso2709Constants.java:23: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/DirectoryBuilder.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/DirectoryBuilder.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/DirectoryBuilder.java:26: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/DirectoryBuilder.java:26: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso646Constants.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso646Constants.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso646Constants.java:25: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso646Constants.java:25: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Label.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Label.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Label.java:25: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Label.java:25: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/LabelBuilder.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/LabelBuilder.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/LabelBuilder.java:26: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/LabelBuilder.java:26: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso646ByteBuffer.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso646ByteBuffer.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso646ByteBuffer.java:31: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Iso646ByteBuffer.java:31: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/FieldHandler.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/FieldHandler.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/FieldHandler.java:23: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/FieldHandler.java:23: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/RecordBuilder.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/RecordBuilder.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/RecordBuilder.java:29: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/RecordBuilder.java:29: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/FieldsBuilder.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/FieldsBuilder.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/FieldsBuilder.java:27: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/FieldsBuilder.java:27: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Record.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Record.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Record.java:28: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/Record.java:28: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/RecordFormat.java:2: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                              ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/RecordFormat.java:2: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * Copyright 2016 Christoph B??hme
58.83                               ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/RecordFormat.java:26: error: unmappable character (0xC3) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                       ^
58.83 /home/playground/metafacture-core/metafacture-biblio/src/main/java/org/metafacture/biblio/iso2709/RecordFormat.java:26: error: unmappable character (0xB6) for encoding US-ASCII
58.83  * @author Christoph B??hme
58.83                        ^
58.83 84 errors
58.83 
58.83 [Incubating] Problems report is available at: file:///home/playground/metafacture-core/build/reports/problems/problems-report.html
58.83 
58.83 FAILURE: Build failed with an exception.
58.83 
58.83 * What went wrong:
58.83 Execution failed for task ':metafacture-biblio:javadoc'.
58.83 > Javadoc generation failed. Generated Javadoc options file (useful for troubleshooting): '/home/playground/metafacture-core/metafacture-biblio/build/tmp/javadoc/javadoc.options'
58.83 
58.83 * Try:
58.83 > Run with --stacktrace option to get the stack trace.
58.83 > Run with --info or --debug option to get more log output.
58.83 > Run with --scan to get full insights.
58.83 > Get more help at https://help.gradle.org.
58.83 
58.83 Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.
58.83 
58.83 You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
58.83 
58.83 For more on this, please refer to https://docs.gradle.org/8.14.4/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
58.83 
58.83 BUILD FAILED in 58s
58.83 9 actionable tasks: 9 executed
------
[+] up 0/1
 ⠙ Image playground-dev-app Building                                                                                                                                                                         202.8s
Dockerfile.dev:47

--------------------

  45 |     # must match dependencies in project.clj

  46 |     #RUN git checkout metafacture-core-8.0.0

  47 | >>> RUN ./gradlew install

  48 |     

  49 |     ## metafacture-playground

--------------------

failed to solve: process "/bin/sh -c ./gradlew install" did not complete successfully: exit code: 1

```

--- a/Dockerfile.dev
+++ b/Dockerfile.dev
@@ -1,28 +1,12 @@
-FROM debian:bookworm-slim
+FROM eclipse-temurin:11-jdk-jammy
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    software-properties-common \
-    wget \
-    gnupg \
+RUN apt-get update && apt-get install -y \
     leiningen \
     git \
-    vim \
     curl \
     unzip \
-    ca-certificates \
-    && apt clean \
-    && rm -rf /var/lib/apt/lists/*
-
-# OpenJDK 11 manuell installieren
-RUN wget https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz \
- && tar xzf openjdk-11.0.2_linux-x64_bin.tar.gz -C /opt \
- && rm openjdk-11.0.2_linux-x64_bin.tar.gz \
- && update-alternatives --install /usr/bin/java java /opt/jdk-11.0.2/bin/java 1 \
- && update-alternatives --install /usr/bin/javac javac /opt/jdk-11.0.2/bin/javac 1
-
-# Node.js 18 installieren über Nodesource
-RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - \
-    && apt-get install -y nodejs \
+    nodejs \
+    npm \
     && rm -rf /var/lib/apt/lists/*
 
 # Use non-root user

--- a/Dockerfile.prod
+++ b/Dockerfile.prod
@@ -1,28 +1,12 @@
-FROM debian:bookworm-slim
+FROM eclipse-temurin:11-jdk-jammy
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    software-properties-common \
-    wget \
-    gnupg \
+RUN apt-get update && apt-get install -y \
     leiningen \
     git \
-    vim \
     curl \
     unzip \
-    ca-certificates \
-    && apt clean \
-    && rm -rf /var/lib/apt/lists/*
-
-# OpenJDK 11 manuell installieren
-RUN wget https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz \
- && tar xzf openjdk-11.0.2_linux-x64_bin.tar.gz -C /opt \
- && rm openjdk-11.0.2_linux-x64_bin.tar.gz \
- && update-alternatives --install /usr/bin/java java /opt/jdk-11.0.2/bin/java 1 \
- && update-alternatives --install /usr/bin/javac javac /opt/jdk-11.0.2/bin/javac 1
-
-# Node.js 18 installieren über Nodesource
-RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - \
-    && apt-get install -y nodejs \
+    nodejs \
+    npm \
     && rm -rf /var/lib/apt/lists/*
 
 # Use non-root user

